pages/linux/grub-mkrelpath.md

```diff
@@ -0,0 +1,16 @@
+# grub-mkrelpath
+
+> Make a filesystem path relative to the root of its filesystem.
+> More information: <https://www.gnu.org/software/grub/manual/grub/html_node/Invoking-grub_002dmkrelpath.html>.
+
+- Set the filesystem path relative to the root which can be used by other GRUB utilities:
+
+`grub-mkrelpath {{path/to/directory}}`
+
+- Display GRUB version number:
+
+`grub-mkrelpath --version`
+
+- Display help:
+
+`grub-mkrelpath --help`
```

Comment:
```suggestion
- Display help:

`grub-mkrelpath --help`

- Display GRUB version number:

`grub-mkrelpath --version`
```

Optional fix: examples can be reordered.
Hint: to find all pages containing `--help` and `--version` you can use `for f in *.md; do if grep -E -- --version "$f" &> /dev/null && grep -E -- help "$f" &> /dev/null; then echo $f; fi; done` one liner.